core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1517,6 +1518,24 @@ public Void visitCorrelatedJoin(CorrelatedJoinNode node, Void context)
             return processChildren(node, context);
         }
 
+        @Override
+        public Void visitTableFunction(TableFunctionNode node, Void context)
+        {
+            NodeRepresentation nodeOutput = addNode(
+                    node,
+                    "TableFunction",
+                    ImmutableMap.of("name", node.getName()));
+
+            checkArgument(
+                    node.getSources().isEmpty() && node.getTableArgumentProperties().isEmpty() && node.getInputDescriptorMappings().isEmpty(),
+                    "Table or descriptor arguments are not yet supported in PlanPrinter");
+
+            node.getArguments().entrySet().stream()
+                    .forEach(entry -> nodeOutput.appendDetails(entry.getKey() + " => " + entry.getValue()));
```

Comment:
We seem to be using `:=` instead of `=>` everywhere else

Reply:
We use `:=` for internal mappings. Here I used `=>` for table function arguments to reflect the argument passing convention. I think it is easier to read if we use the same syntax as in query.